### PR TITLE
NFSAAS-2235 update to RP V3.5 (#7)

### PR DESCRIPTION
**Pull request by leonardbf**
Again. With clean master.
Update SDK to version R3.5 of RP.
Includes active directory and export policy.
Still in preview - breaking changes are not an issue

Swagger:
https://github.com/leonardbf/azure-rest-api-specs/tree/master/specification/netapp/resource-manager

Last swagger PR:
Azure/azure-rest-api-specs#5446

This replaces #5646 

(NetApp internal ticket NFSAAS-2235)

### CODE REVIEW

**Comment by dsgouda** [comment]
@leonardbf props for the clean commit history 👍

File: src/SDKs/NetApp/Management.NetApp/Microsoft.Azure.Management.NetApp.csproj [hunk 1]
```diff
@@ -8,11 +8,17 @@
...
-    <Version>0.9.0-preview</Version>
+    <Version>0.9.1-preview</Version>
     <PackageTags>MicrosoftAzure Management;NetApp</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-        This is a public preview release of the NetApp storage management SDK. It provides the capability to create multiple file systems volumes through Microsoft Azure.
+        Version 0.9.1-preview relates to NetApp Files (ANF) RP R3
+           - Addition of export policy for volumes
+           - Addition of active directory for accounts
+        Version 0.9.0-preview relates to NetApp Files (ANF) RP R2
```
Comment (dsgouda) [inline src/SDKs/NetApp/Management.NetApp/Microsoft.Azure.Management.NetApp.csproj]: nit: You don't need to explain what previous versions of the package do.,

File: src/SDKs/NetApp/NetApp.Tests/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -35,5 +35,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/NetApp/NetApp.Tests/Properties/AssemblyInfo.cs]: Test project versions are always set to `1.0.0.0` and never modified

File: src/SDKs/NetApp/NetApp.Tests/ResourceTests/SnapshotTests.cs [hunk 3]
```diff
@@ -143,8 +143,15 @@ public void PatchSnapshot()
                     Tags = dict
                 };
 
-                var resource = netAppMgmtClient.Snapshots.Update(snapshotPatch, ResourceUtils.resourceGroup, ResourceUtils.accountName1, ResourceUtils.poolName1, ResourceUtils.volumeName1, ResourceUtils.snapshotName1);
-                Assert.True(resource.Tags.ToString().Contains("Tag1") && resource.Tags.ToString().Contains("Value1"));
+                try
```
Comment (dsgouda) [inline src/SDKs/NetApp/NetApp.Tests/ResourceTests/SnapshotTests.cs]: nit: Consider using `Assert.Throws` here

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks great for the most part

File: src/SDKs/NetApp/NetApp.Tests/ResourceTests/SnapshotTests.cs (hunk 3 above)
Comment (leonardbf) [reply src/SDKs/NetApp/NetApp.Tests/ResourceTests/SnapshotTests.cs]: ok. If it's acceptable I will look at this for the next version which is gated GA in the next few weeks.

File: src/SDKs/NetApp/NetApp.Tests/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (leonardbf) [reply src/SDKs/NetApp/NetApp.Tests/Properties/AssemblyInfo.cs]: ok. Updated.

File: src/SDKs/NetApp/Management.NetApp/Microsoft.Azure.Management.NetApp.csproj (hunk 1 above)
Comment (leonardbf) [reply src/SDKs/NetApp/Management.NetApp/Microsoft.Azure.Management.NetApp.csproj]: Ok. I'll keep this in mind but would like to leave it here this time.

**Review by dsgouda — APPROVED** [review]
LGTM will merge when CI builds pass

**Comment by dsgouda** [comment]
@azuresdkci test this please

**Comment by dsgouda** [comment]
@leonardbf please join the azure github org [here](https://aka.ms/azuregithub) in order to kick off this build

**Pull request merged**